types/nkzq__copy-paste/nkzq__copy-paste-tests.ts

```diff
@@ -0,0 +1,13 @@
+import * as CopyPaste from 'copy-paste';
```

Comment:
```suggestion
import * as CopyPaste from '@nkzq/copy-paste';
```
you meant this? 
Also you probably want `paths` property to be set, as this is namespaced package:
https://github.com/DefinitelyTyped/DefinitelyTyped#what-about-scoped-packages